### PR TITLE
add org.gtk.Gtk3theme.Mojave-light

### PR DESCRIPTION
**Pull request by elsiehupp**
> Note: this is a rebased version of https://github.com/flathub/flathub/pull/2183.

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub.
> See: https://github.com/vinceliuice/Mojave-gtk-theme/issues/142#issuecomment-805106552

- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
> This is the weird one. I obviously don't own the domain gtk.org, but the way Flatpak does themes, GTK themes have to start with `org.gtk.Gtk3theme`, and the text immediately after has to match the name of a non-sandboxed theme that has already been installed and activated by the user. I tried using the ID `io.github.elsiehupp.Mojave-dark`, and `appstream-util validate` checked out, but `flatpak-builder` gave me the following error:
> ```
> error: No extension point matching io.github.elsiehupp.Mojave-light in runtime/org.freedesktop.Sdk/x86_64/18.08
> ```
> So, yeah... if this flatpak is ineligible for Flathub, I suppose I could always modify the upstream installer script to obtain the necessary results. For what it's worth, [the Snap Store seems to have been perfectly happy to accept a Snap version of the full set of themes](https://snapcraft.io/mojave-themes), the only problem being that Flatpak apps don't see them.

- [ ] Any additional patches or files have been submitted to the upstream projects concerned.
> The only changes I made were figuring out how to package the theme as a flatpak. Because of the way Flatpak seems to handle themes (in that the name of the theme is part of the ID), and `Mojave-gtk-theme` comprises eight different sub-themes, there isn't really much point in submitting the changes to the upstream since I would be submitting at least eight different flatpaks. If, however, I end up making changes to the flatpak so that it uses the upstream repository directly, that might involve making contributions to the upstream (if only an install script).

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: org.gtk.Gtk3theme.Mojave-light.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "id": "org.gtk.Gtk3theme.Mojave-light",
+  "branch": "3.22",
+  "runtime": "org.freedesktop.Sdk",
+  "build-extension": true,
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "18.08",
```
Comment (hfiguiere) [inline org.gtk.Gtk3theme.Mojave-light.json:7]: I'm pretty sure branch and runtime-version are not right.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: org.gtk.Gtk3theme.Mojave-light.json (hunk 1 above)
Comment (elsiehupp) [reply org.gtk.Gtk3theme.Mojave-light.json:7]: Could you remind me what they should be referring to? It’s been a while since I wrote this. Thanks!

**Comment by hfiguiere** [comment]
> Note: this is a rebased version of #2183.

FYI, you don't need to submit a new PR to rebase.

**Comment by elsiehupp** [comment]
> > Note: this is a rebased version of #2183.
> 
> FYI, you don't need to submit a new PR to rebase.

I rebased onto a different branch (based on the feedback on the other thread), and GitHub automatically closed the other pull request and wouldn’t let me reopen it. I would have used the other PR if GitHub had let me.

File: org.gtk.Gtk3theme.Mojave-light.json (hunk 1 above)
Comment (hfiguiere) [reply org.gtk.Gtk3theme.Mojave-light.json:7]: forget what I said about branch I was incorrect, but `runtime-version` should probably "20.08".

**Comment by hfiguiere** [comment]
ok

File: org.gtk.Gtk3theme.Mojave-light.json (hunk 1 above)
Comment (elsiehupp) [reply org.gtk.Gtk3theme.Mojave-light.json]: Thanks

**Comment by hfiguiere** [comment]
bot, build org.gtk.Gtk3theme.Mojave-light

File: .github/workflows/external-data-checker.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: 'Check if external data changed across Flathub org'
```
Comment (barthalion) [inline .github/workflows/external-data-checker.yml:1]: This and other leftovers from the master branch should be removed. Other than that, LGTM.

**Pull request closed without merging**